.azure-pipelines/js-mgmt-ci.yml

```diff
@@ -0,0 +1,53 @@
+trigger:
```

Comment:
@kpajdzik it doesn't look like you updated DevOps to point at these newly renamed yml files. I was actually expecting the name of the pipeline to be `js - mgmt - ci` not the yml file but I suppose you can call the yml file this as well.

Reply:
I figured it's going to be easier to match those files together.